frontend/package.json

```diff
@@ -18,21 +18,27 @@
   "author": "RMI",
   "license": "MIT",
   "dependencies": {
+    "lucide-react": "^0.511.0",
```

Comment:
Dependency added and used for icons:
<img width="51" alt="Screenshot 2025-05-22 at 16 19 09" src="https://github.com/user-attachments/assets/4af093c0-937d-4579-b1aa-d370a4508b18" />